common/src/View/EntityAttributeGridTable.cpp

```diff
@@ -222,7 +222,10 @@ namespace TrenchBroom {
             if (rowIt != std::end(m_rows)) {
                 rowIt->merge(value, nameMutable, valueMutable);
             } else {
-                const String tooltip = Assets::AttributeDefinition::safeFullDescription(definition);
+                String tooltip = Assets::AttributeDefinition::safeFullDescription(definition);
```

Comment:
I don't think the full description is useful as a tooltip. Maybe the short description is enough.